modlog_wiki_publisher.py

```
@@ -1282,16 +1541,22 @@ def update_wiki_page(reddit, subreddit_name: str, wiki_page: str, content: str,
 
         # Provide more context for common errors
         if "403" in error_str:
-            logger.error(f"403 Forbidden error updating wiki page /r/{subreddit_name}/wiki/{wiki_page}")
+            logger.error(
+                f"403 Forbidden error updating wiki page /r/{subreddit_name}/wiki/{wiki_page}"
+            )
             logger.error("Possible causes:")
```

Ruff (0.15.18)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)